server/src/test/java/org/elasticsearch/cluster/metadata/MetadataIndexTemplateServiceTests.java

```diff
@@ -211,22 +207,11 @@ public void testIndexTemplateWithValidateMapping() throws Exception {
         assertThat(errors.get(0).getMessage(), containsString("analyzer [custom_1] has not been configured in mappings"));
     }
 
-    public void testBrokenMapping() throws Exception {
```

Comment:
With the internal adjustments I made, we wouldn't even get this far and would error out earlier, so I just dropped this test.